QgisModelBaker/gui/drop_message.py

```diff
@@ -62,3 +61,49 @@ def handle_dropped_file_configuration(self, handle_dropped):
             settings.setValue("QgisModelBaker/drop_mode", DropMode.YES.name)
         else:
             settings.setValue("QgisModelBaker/drop_mode", DropMode.NO.name)
+
+
+class DropMessageQuickDialog(QDialog, gui_utils.get_ui_class("drop_quick_message.ui")):
+    def __init__(self, dropped_files, parent=None):
+        QDialog.__init__(self, parent)
+        self.setupUi(self)
+        self.setFixedSize(QSize())
+        QgsGui.instance().enableAutoGeometryRestore(self)
+
+        quick_button = self.button_box.addButton(
+            self.tr("Quickly visualize"), QDialogButtonBox.ActionRole
+        )
+        wizard_button = self.button_box.addButton(
+            self.tr("Go to Model Baker Wizard"), QDialogButtonBox.ActionRole
+        )
+
+        file_list = "\n- ".join(
+            os.path.basename(path)
+            for path in dropped_files
+            if dropped_files.index(path) < 10
+        )
+        self.info_label.setText(
+            self.tr(
+                "Would you like to quickly visualize these files, or would you prefer to create a structured database using the Model Baker Wizard?\n- {} {}"
+            ).format(
+                self.tr("files") if len(dropped_files) > 1 else self.tr("file"),
```

Comment:
Minor suggestion: When multiple files are dropped, it seems not so readable to have the `files` word accompanying the first file base name. Perhaps we could have something like this instead?

```
Files:
- my_file1.xtf
- my_file2.xtf

```

In this way, single file drops would look like this:

```
File:
- my_file1.xtf

```

Which still looks acceptable.

Reply:
The mistake was, that I had not "visualize these {}, or would", because the first parameter is "files" or "file" depending on if there are more than one. Then the list. And then the "and {} more" if there are more than 10. Thanks for pointing on that :-)